trace-sdk/src/main/java/io/bitrise/trace/data/collector/device/DeviceNetworkTypeDataCollector.java

```diff
@@ -80,7 +83,7 @@ private static String getDeviceNetworkType(@NonNull final Context context) {
     @RequiresPermission(android.Manifest.permission.ACCESS_NETWORK_STATE)
     @NonNull
     @Deprecated
-    private static String getNetworkTypeDataFromNetworkInfo(@NonNull final ConnectivityManager connectivityManager) {
+    static String getNetworkTypeDataFromNetworkInfo(@NonNull final ConnectivityManager connectivityManager) {
```

Comment:
L86, L110, L146: did you change the visibility of them just for the tests to use?

Reply:
Yeah. I thought I had added the annotation but it must've got lost somewhere - apologies. I'll put the annotation over these changes.